### PR TITLE
Pset_SpatialZoneTypeCompartment

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_SpatialZoneTypeCompartment

## Description: 

Properties for spatial zones of type compartment

| Property       | Datatype                 | Description                                          |
| -------------- | ------------------------ | ---------------------------------------------------- |
| Material       | IfcLabel                 | Use IfcMaterialDefinition - Not part of the Pset/Qto |
| NominalSpacing | IfcPositiveLengthMeasure | Specifies the nominal spacing between seals          |
| NominalWidth   | IfcPositiveLengthMeasure | Specifies the nominal width of the compartment       |
| NominalHeight  | IfcPositiveLengthMeasure | Specifies the nominal height of the compartment      |

### CODE REVIEW

File: IFC4x3/Properties/n/NominalHeight_0ZM6gGT814qRBtBvJJdu5X/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NominalHeight_0ZM6gGT814qRBtBvJJdu5X/DocProperty.xml:1]: Can we not use the existing NominalHeight?

File: IFC4x3/Properties/n/NominalWidth_1X9otKcjv7p8Guj7jHwQgt/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NominalWidth_1X9otKcjv7p8Guj7jHwQgt/DocProperty.xml:1]: Same question as above. @michel-rives

**Comment by Michel-Rives** [comment]
Yes, pls make use of the existing NominalHeight (of an object).

Thks.  M./

 

De : SergejMuhic ***@***.***> 
Envoyé : jeudi 13 juillet 2023 18:54
À : bSI-InfraRoom/IFC-Specification ***@***.***>
Cc : Michel-Rives ***@***.***>; Mention ***@***.***>
Objet : Re: [bSI-InfraRoom/IFC-Specification] Pset_SpatialZoneTypeCompartment (PR #576)

 

@SergejMuhic commented on this pull request.

  _____  

In IFC4x3/Properties/n/NominalHeight_0ZM6gGT814qRBtBvJJdu5X/DocProperty.xml <https://github.com/bSI-InfraRoom/IFC-Specification/pull/576#discussion_r1241070249> :

> @@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

Can we not use the existing NominalHeight?

  _____  

In IFC4x3/Properties/n/NominalWidth_1X9otKcjv7p8Guj7jHwQgt/DocProperty.xml <https://github.com/bSI-InfraRoom/IFC-Specification/pull/576#discussion_r1241070453> :

> @@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

Same question as above. @Michel-Rives <https://github.com/Michel-Rives>

**Comment by SergejMuhic** [comment]
> Yes, pls make use of the existing NominalHeight (of an object).

Note after the tech meeting, we are keeping the new properties to have the proposed descriptions. The merging can then be discussed with bSI.

**Pull request merged**